src/hooks/useOttAnalytics.ts

```diff
@@ -2,9 +2,15 @@ import { useCallback, useEffect, useState } from 'react';
 
 import type { PlaylistItem } from '#types/playlist';
 import { useConfigStore } from '#src/stores/ConfigStore';
+import { useAccountStore } from '#src/stores/AccountStore';
 
 const useOttAnalytics = (item?: PlaylistItem, feedId: string = '') => {
   const analyticsToken = useConfigStore((s) => s.config.analyticsToken);
+  const user = useAccountStore((state) => state.user);
+
+  // oaid (Ott App User ID) for analytics
+  const oaid: number | undefined = Number(user?.id);
```

Comment:
Should we only send it for JW subscription model?

Reply:
I added a check for it 🙂